documents/int256.go

```diff
@@ -129,3 +129,25 @@ func isValidInt256(n big.Int) bool {
 
 	return true
 }
+
+// Add sets i to the sum x+y and returns i
+func (i *Int256) Add(x *Int256, y *Int256) *Int256 {
```

Comment:
this should check for the overflow. I think you can re-use the function called `validInt256` and return an error if not